multitemporal/bin/example.py

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env python
 
+from __future__ import print_function
```

Comment:
Don't need this anymore right?

Reply:
If memory serves, that was put in by futurize, which tries to support both python 2 and 3.

Reply:
Ohh, sorry, I forgot I set the thing up to support both 2 and 3 when I added in `python_requires='>=3'`.  I shouldn't've let this PR sit for 3 weeks because that increases the risk of forgetting stuff.  :disappointed:

Anyway the line's harmless and roughly idiomatic for code that's been ported from 2 to 3 so the principle of maximizing the work not done suggests we leave it.

However I care about getting things merged more than I care about any other principle.  Let me know.

Reply:
I take @bhbraswell's :+1: to mean this is resolved.